### PR TITLE
Updates to vscode-language libraries

### LINKED ISSUE #1357: Update vscode-language* libraries
Per https://github.com/microsoft/vscode-cosmosdb/issues/1334#issuecomment-596407878:

> And you are using pretty outdated version of the libraries:
> 
> ```
>         "vscode-languageclient": "^4.4.0",
>         "vscode-languageserver": "^4.4.0",
> ```
> 
> The onReady handling has majorly changed in the 6.x version to address bugs around reconnects.

### PR DESCRIPTION
**Pull request by neelip**
Fixes #1357

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -60,7 +60,8 @@ let config = dev.getDefaultWebpackConfig({
         new CopyWebpackPlugin([
             // getCoreNodeModule.js -> dist/node_modules/getCoreNodeModule.js
             { from: './out/src/utils/getCoreNodeModule.js', to: 'node_modules' },
-        ]),
```
Comment (nturinski) [inline webpack.config.js:63]: Haha how did this change happen?

**Review by nturinski — COMMENTED** [review]
I did some testing with the Mongo scrapbook language service stuff and everything seems fine, as well as creating/deleting Mongo accoutns/docs/collections.  Also don't see the "Language client is not ready yet" error in the Mongo Language Service output.

I wanna confirm with @EricJizbaMSFT, but that's the primary reason for upgrading the vscode-language libraries, right?

File: webpack.config.js (hunk 1 above)
Comment (neelip) [reply webpack.config.js:63]: I was testing some in this file and missed this change. Thanks for catching.

**Review by nturinski — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1058,8 +1059,9 @@
         "underscore": "^1.8.3",
         "vscode-azureextensionui": "^0.36.0",
         "vscode-json-languageservice": "^3.0.8",
-        "vscode-languageclient": "^4.4.0",
-        "vscode-languageserver": "^4.4.0",
+        "vscode-languageclient": "^6.1.3",
```
Comment (EricJizbaMSFT) [inline package.json:1062]: Double check the minimum version of VS Code required for these and make sure ours matches it

Comment (neelip) [reply package.json:1062]: Thank you! Checked the versions it looks good. 
`vscode-languageclient@6.1.3` requires ^1.41.0 version of VSCode engine, current version of VSCode is at 1.48.0
https://github.com/microsoft/vscode-languageserver-node/commit/20a60b0efb23516dd6c4edfedb72d8729c85cdae

**Pull request merged**